### DISCUSSION TITLE
Changing "Next" button in Wizard

### DISCUSSION THREAD

**Discussion by kvostyc** [filamentphp/filament#6766] [Help]
Hey, 

im wondering if there is a way how to change text or whole component in Wizard step

Im attaching image to be clear
![obrázok](https://github.com/filamentphp/filament/assets/90652228/4519f7cc-5307-46cc-82c9-7b8b8a059fbd)

**Comment by kvostyc** [comment]
Is there a way how to change that ?

**Reply by pboivin** [reply]
Not sure, but I think the only way is to publish the translations

**Reply by kvostyc** [reply]
Will definitely try that, but it would be better if u just could set it with some method ngl, maybe i should create PR for that :D

**Comment by emmexx** [comment]
In v3 you can use the methods nextAction() e previousAction():
[Customizing the wizard action objects](https://filamentphp.com/docs/3.x/forms/layout/wizard#customizing-the-wizard-action-objects)

**Reply by ashkesarmaa** [reply]
Yeah but is there any way to change the next button label only for a specific step?

**Comment by 8888post** [comment]
```
Wizard::make([
    // ...
])->nextAction(
        fn (Action $action) => $action->label('Next step'),
    )
```

**Comment by emjay64** [comment]
Can it be changed on `Action  ->steps()`

**Reply by slizhva** [reply]
```php
Action   
->steps($schema);
->modifyWizardUsing(
    fn(Wizard $wizard) => $wizard->nextAction(fn(Action $action) => $action->label('Save & Next')),
);
```
